contracts/Router.sol

```diff
@@ -239,6 +239,11 @@ contract Router is ReentrancyGuard {
 	    tree.transferFrom(msg.sender, address(this), amountIn);
 
 	    if (totalInBurnPool > 0) {
+            // To find how much of the reserve is available to distribute, start with the reserve balance in this
+            // contract, subtract the total that was just pledged, subtract the reserve that is waiting to be claimed
+            // from previous burns. Multiply this by the square root of the percentage of TREE in the burn pool.
+            // For example, if 49% of the TREE supply is being burned, 70% (sqrt .49 = .70) of the available reserve
```

Comment:
Good example :+1:

Reply:
Except I realized the original TREE math used quadratic burning to be [0.49 * 0.49 or 0.2401](https://github.com/WhalerDAO/tree-contracts/blob/4525d20def8fce41985f0711e9b742a0f3c0d30b/contracts/TREEReserve.sol#L197). My concern is this would drain the reserve quicker than we expect if we no longer go with quadratic burning

Reply:
Pasting my comment to you from discord:

I thought of the same thing.  The main thing countering this is that burning requires a rebase.  I purposely subtracted the new reserve pledged from the burn distribution so there will always be something in the reserve.

uint reserveToDistribute =
               (reserveToken.balanceOf(address(this)).**sub(totalPledged)**.sub(totalReserveClaimable))

The dynamic is changing from almost no burning to a reserve that gets distributed and replenished frequently.(edited)

But it requires rebases.  Rebases are the lifeblood of TREE.

If you think about it, most of our reserve came from just the first two rebases.  What we want to recreate is lots of action and big, frequent rebases.

Then you burn again if you want to get at what was put in the reserve from the last rebase, but to burn, you also need to trigger a rebase.  Combine that with a slowly rising target price, and you get a never ending cycle.